### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by flore77**
## Need

```gherkin
As a developer
I need to have some integration tests
So I will know for sure that may app functions well
```

## Deliverables
Tests to ensure that Mugshot is running comply the specs.

## Solution

Same tests like in the [Chai-Mugshot](https://github.com/uberVU/chai-mugshot/tree/4ed55c774817a0dbfea993cf4045d66008f18638/test) plugin.

### CODE REVIEW

**Comment by flore77** [comment]
@andrei-picus-hs please review the code

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   },
   "scripts": {
     "test": "mocha",
```
Comment (andrei-picus-hs) [inline package.json]: 1. Update this.
2. Add a `test-integration` script that runs the integration tests.

File: test/integration/test.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```
Comment (andrei-picus-hs) [inline test/integration/test.html]: Rename the `test` folder to `tests`.

File: package.json [hunk 3]
```diff
@@ -23,6 +23,7 @@
   },
```
Comment (andrei-picus-hs) [inline package.json]: Add `webdriverio` as a dev dep. Pro tip: try to avoid installing non-CLI packages globally on your host.

**Comment by andrei-picus-hs** [comment]
Finished review.

File: package.json (hunk 3 above)
Comment (flore77) [inline package.json]: Yes, thanks. It is not installed globally, I have installed it in the `node_modules` folder to make some local tests. And now I forgot to include it in the devDependencies.

**Comment by flore77** [comment]
@andrei-picus-hs please review changes

File: package.json [hunk 4]
```diff
@@ -19,10 +19,13 @@
     "mocha": "^2.2.5",
     "pre-git": "^0.6.2",
     "sinon": "^1.15.4",
-    "sinon-chai": "^2.8.0"
+    "sinon-chai": "^2.8.0",
+    "webdriverio": "^3.2.0"
   },
   "scripts": {
-    "test": "mocha",
+    "test": "mocha tests/unit tests/unit/differ",
```
Comment (andrei-picus-hs) [inline package.json]: Use `--recursive`.

**Comment by andrei-picus-hs** [comment]
LGTM

**Pull request merged**